guide/blueprints/yaml-reference.md

```diff
@@ -281,3 +281,43 @@ elements for more information:
 * `DynamicCluster`: provides resizable clusters given a `dynamiccluster.memberspec` set with `$brooklyn.entitySpec(Map)` as described above 
 * `DynamicFabric`: provides a set of homogeneous instances started in different locations,
   with an effector to `addLocation`, i.e. add a new instance in a given location, at runtime
+
+## Notable Tags
+
+Some tags are used by convention and in the UI for special purposes.
+These are:
+
+* `ui-composer-annotation`: text (interpreted as markdown without HTML support) which will be displayed on a node in the blueprint composer,
+  or a map containing a key with the `text` and optionally any/all of `{x, y, width, height, background, style, styleInnerDiv}` for displaying it.
+For example:
+
+  ~~~ yaml
+  name: Annotation Sample
+  services:
+  - type: server
+    brooklyn.tags:
+      - ui-composer-annotation: A simple default annotation
+      - ui-composer-annotation:
+          text: >-
+            Shown below, yellow text, centered with CSS. Because it's long, scroll bars horizontally and vertically shown when
+            needed.
+          styleInnerDiv: 'margin: auto; color: yellow;'
+          'y': 120
```

Comment:
Do we need quotes for `y`?

Reply:
not sure why, but the JS yaml processor puts these in.  probably it wants to treat `y` like a reserved word for `yes` aka `true`.  i just cut and pasted.  but agree, worth removing here.